pagopa/io-app
# chore: [IOBP-906] Add interrupted payment screen outcome

## Short description
This PR adds the interrupted payment screen that will be shown to the user when the in-app browser during a payment is dismissed;

## List of changes proposed in this pull request
- Added the additional outcome `24` as `IN_APP_BROWSER_CLOSED_BY_USER`;
- Added locales In IT/EN lang;
- Mapped the outcome screen with the title and subtitle showing the correct text;

## How to test
- Start a payment flow from "Payments" section by tapping the CTA "Pay a notice";
- Input a notice code and then the fiscal code (if you're in local env, enter any value until you reach the maximum character limit);
- Go forward by pressing the CTA in that screen until you reach the confirmation screen where there is the "Pay XX,XX€" CTA;
- Tap on that CTA, there must show an in-app browser, then tap on the "Close" button

## Preview

https://github.com/user-attachments/assets/5d61a29d-b1c2-44e6-b8e9-72bb4c5f8f7d
